PSReadLine/Accessibility.cs

```diff
@@ -10,14 +11,73 @@ internal class Accessibility
     {
         internal static bool IsScreenReaderActive()
         {
-            bool returnValue = false;
-
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                return IsAnyWindowsScreenReaderEnabled();           // Check for macOS VoiceOver

```

Comment:
The existing style in this repo always uses curly braces for if blocks, even for the single-statement if blocks. Can you please update the code to adhere to the existing style? Thanks

Reply:
It doesn't though, I checked 🫠

Reply:
Let's add a formatting tool if there's a consistent style we're wishing to apply to this repo. Prettier is a great tool.

Reply:
> It doesn't though, I checked

My bad :) About this style, it was mixed originally in the repo. After it was transferred to PowerShell org, I followed the style used in PowerShell code ([`csharp_prefer_braces = true:silent`](https://github.com/PowerShell/PowerShell/blob/master/.editorconfig#L134) -- prefer to include braces even for single-statement blocks).

It was only me working in the repo before, so formatting was not a problem :) Totally agree formatting tools will help in this case.